pages/index.vue

```diff
@@ -532,7 +532,7 @@
         <section>
           <SmartTabs>
             <SmartTab :id="'history'" :label="$t('history')" :selected="true">
-              <HttpHistory @useHistory="handleUseHistory" ref="historyComponent" />
+              <History v-bind:page="'rest'" @useHistory="handleUseHistory" ref="historyComponent" />
```

Comment:
```suggestion
              <History :page="'rest'" @useHistory="handleUseHistory" ref="historyComponent" />
```
Let's stick with using shorthands for binding props